classes/Feed.php

```diff
@@ -153,15 +153,16 @@ public function optionsFromDefault(?Pages $pages = null, $options = []): array
         $options['items'] = $items;
         $options['link'] = url($options['link']);
 
+        $modified = $items->first()->modified($options['dateformat'], 'date');
```

Comment:
the `if ($items && $items->count()` below is necessary. in case the list is empty